unlock-app/src/components/interface/checkout/alpha/Connected.tsx

```diff
@@ -63,11 +64,13 @@ interface SignedOutProps {
     provider: 'METAMASK' | 'UNLOCK' | 'WALLET_CONNECT' | 'COINBASE'
   ): Promise<void>
   onUnlockAccount(): void
+  injectedProvider: WalletLinkProvider
```

Comment:
Not sure this is the correct type?

Reply:
changed to any because I don't see any type which we use for this. We `any` in the window object.